pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/ActivityInstanceDto.java

```diff
@@ -1,18 +1,22 @@
 package org.broadinstitute.ddp.db.dto;
 
+import org.broadinstitute.ddp.model.activity.types.ActivityType;
+import org.broadinstitute.ddp.model.activity.types.InstanceStatusType;
 import org.jdbi.v3.core.mapper.reflect.ColumnName;
 import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
 
 public class ActivityInstanceDto {
 
     private long id;
     private String guid;
+    private long studyId;
     private long activityId;
     private long participantId;
     private long createdAtMillis;
     private boolean isReadonly;
     private boolean isHidden;
-    private String statusTypeCode;
+    private InstanceStatusType statusType;
```

Comment:
We should prefer enums over strings for types.

Reply:
I agree